src/picongpu/submit/taurus-tud/Slurm_Tutorial.rst

```diff
@@ -36,4 +36,7 @@ Job Control
 
   * `scancel 12345` abort job
   * `scancel -s Number 12345` send signal or signal name to job
-  * `scontrol update timelimit=4:00:00 jobid=12345`
+  * `scontrol update timelimit=4:00:00 jobid=12345` change the walltime of the job
+  * `scontrol update jobid=12345  dependency=afterany:54321` only start the job after `54321` has finished
+  * `scontrol hold jobid=12345 ` prevent the job from staring
+  * `scontrol unhold jobid=12345 ` allow the job to start (after a hold was set)
```

Comment:
change comment to: release the job to be eligible for run (after it was set on hold)